bootstrap.d/dev-libs.yml

```diff
@@ -627,8 +625,9 @@ packages:
         - host-gettext
       regenerate:
         - args: ['./gitsub.sh', 'pull']
-        # Gnulib broke on commit e3174b6d1fdbe6ea2297bf8c8333f65f9d9d9588, so check out the one before that.
-        - args: ['git', 'checkout', '766ec17a90f67e8cda78394e58a7fffb00f5a4b7']
+        # Checkout current gnulib master, 044bf893acee0a55b22b4be0ede0e3ce010c480a, 29-08-2022.
+        # This avoids random breakage in the future
+        - args: ['git', 'checkout', '044bf893acee0a55b22b4be0ede0e3ce010c480a']
```

Comment:
This also means that we won't get updated automatically though.

Reply:
Is it feasible to add gnulib as a source, so that we can use a local copy of gnulib that we can also update regularly?

Reply:
that would be preferable

Reply:
While preferred, unlikely, as different packages want different versions.

Reply:
python is backwards compatible

Reply:
oh, my bad, that's for the other comment threat; are you sure newest version doesn't work for all?

Reply:
I don't remember whether I asked this before, but I think I did: does something wanting gnulib X break with gnulib Y (when Y>X)?

Reply:
It can and has done so in the past iirc.

Reply:
in that case, seems more reasonable to stick to dist tarballs for anything that uses gnulib, but that's off topic for this PR

Reply:
Yeah I was looking at that, but out of scope for this PR.